docs/getting_started/ai-resources.md

```diff
@@ -0,0 +1,73 @@
+# AI Trainings and Resources Guide
+
+## CU Boulder & CURC Resources
+
+### CU Boulder: AI Efforts on Campus
+- Overview of AI initiatives and support across campus. [https://www.colorado.edu/information-technology/ai-cu-boulder/ai-efforts-campus](https://www.colorado.edu/information-technology/ai-cu-boulder/ai-efforts-campus)
+### RMACC-CU AI Workshop
+- Workshops held in partnership with the Rocky Mountain Advanced Computing Consortium. [https://www.colorado.edu/rc/cu-rmacc-ai-workshop](https://www.colorado.edu/rc/cu-rmacc-ai-workshop)
+### CURC AI/ML Trainings
+- CURC offers trainings each semester to help researchers deploy and optimize AI/ML workflows.
+    * [Setting up LLMs on CURC Resources (an RC Short Course)](https://github.com/ResearchComputing/setting_up_llms_on_curc_resources_rc_short_course)
+    * [Applied Containerization for Machine Learning in HPC (an RC Short Course)](https://colorado.libcal.com/calendar/events/contain)
+- For a complete list of current semester offerings, see [CURC Trainings Page](../getting_started/current-sem-trainings.md)
+### CURC AI/ML Software Stack
+- The following AI/ML software are available and actively supported on CURC systems:
+    * PyTorch (1.13.0): Popular deep learning framework supporting GPU acceleration.
+    * AlphaFold: An AI tool from DeepMind that predicts 3D protein structures from amino acid sequences, revolutionizing biological and medical research. See documentation and usage details [here](../software/alphafold.md). 
+    * CUDA Toolkit, cuDNN, NVIDIA HPC SDK: For GPU-accelerated AI training and inference.
+- Additional supported software can be found in the [CURC Software List](../software/curc_provided_software.md) or accessed via the `module avail` command.
```

Comment:
I would probably remove the "or accessed via the `module avail` command." part. If you want to keep this, then `module spider` would be more appropriate as the `module avail` command might hide things.